.github/workflows/ci-actions.yml

```diff
@@ -58,8 +86,9 @@ jobs:
     needs: build-jdk8
     timeout-minutes: 120
     strategy:
+      fail-fast: false
```

Comment:
-1 on changing tihs. It will hurt our CI bandwidth and doesn't give you any new information

Reply:
That's useful information. Because knowing it only fails on a particular JDK is useful.

Right now, the bandwidth is more hurt because we run all the native tests even if the JDK 8 fails.

Reply:
Or we should go back to our previous set up being we build on JDK 8 first and then we build the rest.

But not having a clue if it's an issue specific to one JDK will be a problem.

Reply:
@n1hility so I only disabled fail fast for forked repo. That way, we fail fast in the main repo and in their own repo, people have the full feedback.